packages/cdk8s-cli/src/util.ts

```diff
@@ -32,6 +32,35 @@ export async function mkdtemp(closure: (dir: string) => Promise<void>) {
   }
 }
 
+export async function callLibrary(command: string, outdir: string) {
```

Comment:
Rename to `synthApp` and combine with `validateSynthesis`

Reply:
Can you please also write a small unit test for this?

Reply:
I wrote a couple of example tests, if you think that's the kind of style we want there I can fill them out a bit more!